packages/vaults/src/transaction/transfer/transfer_icrc1_quorum.ts

```diff
@@ -9,14 +9,41 @@ import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 import {Principal} from "@dfinity/principal";
 
-
+/**
+ * Interface for a transaction that transfers ICRC-1 tokens.
+ * Requires amount of approvals equal to quorum.
+ * This transaction can be created or approved only by an admin.
```

Comment:
is it? member role cannot transfer ICRC-1 tokens?

Reply:
Nope